src/ext/modules/routines.coffee

```diff
@@ -614,14 +625,13 @@ exports.init = (_) ->
     transforms
 
   inspectObject = (inspections, name, origin, obj) ->
+    blacklistedAttributes = blacklistedAttributesSchemaIndependent
+
     schemaType = obj.__meta?.schema_type
-    blacklistedAttributes = if schemaType
+    if schemaType
```

Comment:
will this add merge blacklistedAttributesSchemaIndependent and blacklistedAttributesBySchema[schemaType] in case schema is present? I don't think so

Reply:
it will, I tested. Firstly, independent are saved in blacklistedAttributes, and then in case schema is present, schema are added one by one to blacklistedAttributes (which already contains independent). and in case that independent is blacklisting some attribute and schema is blacklisting the same attribute, independent will be "overriden" by schema so there will be no duplicates